src/Raven.Client/Documents/Session/Tokens/OrderByToken.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Text;
+using Raven.Client.Documents.Queries;
+
+namespace Raven.Client.Documents.Session.Tokens
+{
+    public class OrderByToken : QueryToken
```

Comment:
I'm worried about the number of instances that this might generate during this, no?